src/ui/public/vis/editors/components/editor_options_group.js

```diff
@@ -38,6 +40,39 @@ function EditorOptionsGroup(props) {
       </EuiAccordion>
     </EuiPanel>
   );
+
+  const simpleOptionGroup = (
+    <EuiPanel
+      grow={props.grow}
+      className={`editorOptionsGroup__panel ${props.className}`}
+    >
+      <EuiFlexGroup direction="column">
+        <EuiFlexItem grow={false}>
+          <EuiFlexGroup>
+            <EuiFlexItem>
+              <EuiTitle size="xs">
+                <h2>{props.title}</h2>
+              </EuiTitle>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              {props.actions}
+            </EuiFlexItem>
+          </EuiFlexGroup>
+
+          <EuiSpacer size="m"/>
```

Comment:
Panel layouts it's content different than accordion. Looking in the rendered result, I would suggest that we remove the `EuiSpacer` in this case, since it would introduce noticeable more spacing than the `EuiAccordion` variant would have.